flink-python/src/main/java/org/apache/flink/python/env/AbstractPythonEnvironmentManager.java

```diff
@@ -101,9 +104,11 @@ public AbstractPythonEnvironmentManager(
             PythonDependencyInfo dependencyInfo,
             String[] tmpDirectories,
             Map<String, String> systemEnv,
-            JobID jobID) {
+            JobID jobID,
+            Configuration config) {
         this.dependencyInfo = Objects.requireNonNull(dependencyInfo);
         this.tmpDirectories = Objects.requireNonNull(tmpDirectories);
+        this.systemEnvEnabled = config.get(PythonOptions.PYTHON_SYSTEMENV_ENABLED);
```

Comment:
We can move this to the `createPythonEnvironmentManager` so that we can decide whether systemEnv is a empty Map